### PR TITLE
Implement Tweets List

### PR DESCRIPTION
**Pull request by anastasiialisikh**

### CODE REVIEW

File: src/components/CreateTweet/index.tsx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import CreateTweet from './CreateTweet';
```
Comment (stanimirtt) [inline src/components/CreateTweet/index.tsx]: Please change that to `export { CreateTweet} from...`

File: src/components/TweetItem/index.tsx [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import TweetItem from './TweetItem';
```
Comment (stanimirtt) [inline src/components/TweetItem/index.tsx]: Here too.

File: src/components/TweetList/TweetList.tsx [hunk 3]
```diff
@@ -1,9 +1,57 @@
...
+  constructor(props: IAllTweetsProps) {
+    super(props);
+  }
+
+  getAllPosts(): Promise<ITweet[] > {
+    const fetch = Fetch.request(env.securedRoutes + '/posts', {
+      method: 'GET'
+    });
+
+  return fetch;
```
Comment (stanimirtt) [inline src/components/TweetList/TweetList.tsx]: Add space here.

File: src/components/TweetList/TweetList.tsx [hunk 4]
```diff
@@ -1,9 +1,57 @@
...
+import { TweetItem } from '../TweetItem';
+import { CreateTweet } from '../CreateTweet';
+
+export interface IAllTweetsProps {
+  stores?: IStores
+}
+
+@inject("stores")
+@observer
+class TweetList extends React.Component<IAllTweetsProps, {}> {
```
Comment (stanimirtt) [inline src/components/TweetList/TweetList.tsx]: Use `export class ...` directly here.

File: src/interfaces/tweet.ts [hunk 5]
```diff
@@ -0,0 +1,8 @@
+export interface ITweet {
+  authorId: number;
+  content: string;
+  id?: number;
+  commentsCount?: number;
+  likesCount?: number;
+  user?: any;
+  }
```
Comment (stanimirtt) [inline src/interfaces/tweet.ts]: Remove one space here.

**Review by stanimirtt — APPROVED** [review]

**Review by dpyskunova — APPROVED** [review]

File: src/components/TweetList/TweetList.tsx [hunk 6]
```diff
@@ -1,9 +1,55 @@
...
+
+@inject("stores")
+@observer
+export class TweetList extends React.Component<IAllTweetsProps, {}> {
+  constructor(props: IAllTweetsProps) {
+    super(props);
+  }
+
+  getAllPosts(): Promise<ITweet[] > {
+    const fetch = Fetch.request(env.securedRoutes + '/posts', {
```
Comment (fstodulski) [inline src/components/TweetList/TweetList.tsx]: we could use
`return Fetch.request(******......`
instead returning const with it

**Review by fstodulski — APPROVED** [review]
Perfection

**Review by suezzo — APPROVED** [review]

**Pull request merged**